### PR TITLE
Chore: camera reel analytics and tilt

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

- add all analytics, but not photo manipulation that comes from the context menu (delete, share, etc)
- add tilt behaviour (q/e) for the camera movement
- add controller support
- fixes placeApi exception when photo was taken outside of GenesisCity (on the map borders - sea view)
- sidebarUI adjustment by UX (@RominaMarchetti)

## How to test the changes?

1. Launch the explorer with respective CLI args
2. Enable InWorld Camera mode
3. Use q/e for tilting
4. Verify tilting is reset each time you lock/unlock mouse
5. Try to control camera with Gamepad (all gamepad buttons/controls are used, discover them). Gamepad aiming is only working when mouse is locked
6. Try to make picture on the border of the Genesis City - it should work as usual photo

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/InWorldCamera/InWorldCamera/ScreenshotMetadataBuilder.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public async UniTask BuildAsync(CancellationToken ct)
 
             PlacesData.PlaceInfo? placeInfo = await placesAPIService.GetPlaceAsync(at, ct);
 
-            return (placeInfo?.title ?? "Unknown place", placeInfo.id);
+            return placeInfo == null ? ("Unknown place", "Unknown place") : (placeInfo.title, placeInfo.id);
```
Comment (pravusjif) [inline Explorer/Assets/DCL/InWorldCamera/InWorldCamera/ScreenshotMetadataBuilder.cs:86]: I guess a micro optimization could be having the "Unknown place" (or even the whole `("Unknown place", "Unknown place")`) in a `const`, although maybe the compiler is already smart enought to re-use it the way you already have it today.

**Review by pravusjif — APPROVED** [review]
LGTM! great job man!

File: Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsConfiguration.cs [hunk 2]
```diff
@@ -102,20 +102,20 @@ private bool TryCreateSegmentConfiguration(out Configuration configuration)
...
                 eventToggles = new Dictionary<string, AnalyticsEventToggle>();
 
-                foreach (AnalyticsGroup group in groups)
-                foreach (AnalyticsEventToggle eventToggle in group.events)
-                    eventToggles.Add(eventToggle.eventName, eventToggle);
-            }
-
-            return eventToggles.TryGetValue(eventName, out AnalyticsEventToggle toggle) && toggle.isEnabled;
+            foreach (AnalyticsGroup group in groups)
+            foreach (AnalyticsEventToggle eventToggle in group.events)
```
Comment (AnsisMalins) [inline Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsConfiguration.cs:111]: Can I get an indent here?

**Review by AnsisMalins — APPROVED** [review]
I cannot be sure if the approach is correct, but assuming it is, looks fine.

File: Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsConfiguration.cs (hunk 2 above)
Comment (popuz) [reply Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsConfiguration.cs:111]: It is matter of taste and team agreement on style. I think in this case it reads quite well without indentation, but I will add it for you.

**Pull request merged**